client/src/components/ranking/RankingInfo.js

```diff
@@ -0,0 +1,28 @@
+import '../../styles/Popup.css';
+import Popup from 'reactjs-popup';
+
+const RankingInfo = () => {
+  return (
+    <Popup trigger={
+      <button className='info-icon-button'>
+        <h2 className='info-icon'>i</h2>
+      </button>
+    }
+    modal nested
+    >
```

Comment:
Styling suggestions: could these > characters be moved to the end of the previous line? There are a couple of them in other files as well.

Reply:
Changed the ones I could find